specs/test_5.js

```diff
@@ -0,0 +1,14 @@
+let MainPage = require("../pages/main.page");
+let SearchResultsPage = require("../pages/searchResults.page");
+
+describe('Hotline Protractor Test - Search suite', () => {
+    it('TC #5. Search results', async () => {
+        await MainPage.open();
+        await MainPage.waitForPageToBeAvailable();
+        await MainPage.search('samsung galaxy');
+        await SearchResultsPage.waitForSearchPageToBeAvailable();
+        await SearchResultsPage.getSearchResults();
```

Comment:
Method `getSearchResults()` should be refactored

Reply:
done